tests/by-util/test_tr.rs

```diff
@@ -1313,3 +1313,14 @@ fn check_regression_class_blank() {
         .no_stderr()
         .stdout_only("a12b");
 }
+
+// Check regression found in https://github.com/uutils/coreutils/issues/6163
+#[test]
+fn check_regression_issue_6163() {
+    new_ucmd!()
+        .args(&["-c", "-t", "Y", "Z"])
+        .pipe_in("X\n")
+        .succeeds()
+        .no_stderr()
+        .stdout_only("X\n");
+}
```

Comment:
You mentioned `print '\0' | tr -c -t Y Z` as another testcase in a comment. Could you add this to the testsuite? Or perhaps I missed it and it already exists?

Reply:
I have added them now c: